modules/search/class.jetpack-search-template-tags.php

```diff
@@ -135,6 +153,89 @@ public static function render_filter( $filter, $default_post_types ) {
 		<?php
 	}
 
+	/**
+	 * Renders a single filter for instant search.
+	 *
+	 * @since 8.3.0
+	 *
+	 * @param array $filter             The filter to render.
+	 */
+	public static function render_instant_filter( $filter ) {
+		if ( empty( $filter ) || empty( $filter['buckets'] ) ) {
```

Comment:
is empty check appropriate here or should it be an array check? 169,171 has it as an array.

Reply:
cc @gibrown

Reply:
I think whether it is not an array, or if it is an empty array that the result is the same and we don't have anything to display.